buildRtl/RTL-Config.json

```diff
@@ -0,0 +1,28 @@
+{
+	"multiPass": "dappnode",
+	"port": "80",
+	"host": "0.0.0.0",
+	"SSO": {
+		"rtlSSO": 0
+	},
+	"nodes": [
+		{
+			"index": 1,
+			"lnNode": "Node 1",
+			"lnImplementation": "LND",
+			"Authentication": {
+				"macaroonPath": "/shared"
+			},
+			"Settings": {
+				"userPersona": "MERCHANT",
+				"themeMode": "NIGHT",
+				"themeColor": "PURPLE",
+				"channelBackupPath": "/shared/lnd/backup",
+				"enableLogging": "true",
+				"fiatConversion": "true",
+				"currencyUnit": "USD",
+				"lnServerUrl": "https://lnd.lightning-network.public.dappnode:8080"
+			}
+		}
+	]
+}
```

Comment:
If the file is templated below in the entrypoint why is it here too?

Reply:
For possible different approaches in the future. But I will remove it